taxi/models.py

```diff
@@ -0,0 +1,41 @@
+from django.contrib.auth.models import AbstractUser
```

Comment:
You have imported `AbstractUser` from `django.contrib.auth.models`. But it's better to use `AUTH_USER_MODEL` from settings as it's more flexible and allows changing the user model globally if needed in the future.